src/cljs/bluegenes/core.cljs

```diff
@@ -26,22 +25,24 @@
 ;    (println "dev mode")))
 
 (defn mount-root []
+  (re-frame/clear-subscription-cache!)
+  (route/init-routes!)
   (reagent/render [views/main-panel]
                   (ocall js/document "getElementById" "app")))
 
-(defn navigate-to-deep-links []
-  (let [url (oget js/window "location" "hash")
-        hashless-path (last (clojure.string/split url #"#"))]
-    (cond (> (count url) 2) ;; if there is more than #/ in the url, navigate there
-          (if (= (first hashless-path) "/")
-            (navigate! hashless-path)
-            (navigate! (str "/" hashless-path))))))
+;; TODO replace or delete this function depending on needs with reitit
```

Comment:
what are the considerations here with regards to the todo?

Reply:
Deep linking seems to be working fine without this, so we can delete this code.